src/main/groovy/life/qbic/oncostore/parser/MetadataContext.groovy

```diff
@@ -8,13 +8,15 @@ import life.qbic.oncostore.model.Sample
 
 class MetadataContext {
 
-    final boolean isSomatic
-    final VariantCaller variantCalling
-    final Annotation variantAnnotation
-    final ReferenceGenome referenceGenome
-    final Case patient
-    final Sample sample
-    final List<String> vcfFiles
+    boolean isSomatic
+    VariantCaller variantCalling
+    Annotation variantAnnotation
+    ReferenceGenome referenceGenome
+    Case patient
+    Sample sample
+    List<String> vcfFiles
```

Comment:
Why did you change the properties to be not final anymore do you expect a change?

Reply:
Thanks for the feedback @KochTobi ! As far as I remember, the reason was that I'm creating a new metadata context in `VariantstoreInformationCenter` during variant export in order to store/provide the reference genome and annotation software information. Would you prefer a constructor with these two properties?

Reply:
That's a question on what the metadata context is supposed to provide. mandatory metadata should be final but optional could also be non-final. Since in the case of the VariantStoreInformationCenter you only use it to store `referenceGenome` and `annotationSoftware` which you always expect to be there maybe consider changing the method return from `[variants, metadata]` to `[variants, referenceGenomes, annotationSoftware]`. I also noticed that you do only retrieve information from the first variant. Can you be sure that this information is the same across all detected variants found by your query?

Reply:
I have to think about the first part a bit ;)
>  I also noticed that you do only retrieve information from the first variant. Can you be sure that this information is the same across all detected variants found by your query?

Good point. I did it like that for now but in the end you cannot be sure. It's a bit tricky because theoretically you could have variants in the store that have been annotated with different annotation software and based on different reference genomes. In the end it wouldn't make sense to have these variants in the same exported VCF file. 

Now I'm think if it might be better to make `referenceGenomes` and `annotationSoftware` mandatory query parameters (with default values) for the VCF export.

Reply:
That might be a solution for making sure you export the correct variants/information.

Reply:
I adapted it accordingly. Would be cool if you can have a look and let me know what you think 👍